src/System.Net.Security/tests/UnitTests/System.Net.Security.Unit.Tests.csproj

```diff
@@ -57,4 +57,13 @@
     </Compile>
   </ItemGroup>
   <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.targets))\dir.targets" />
+  <Target Name="RemoveProductionReference" BeforeTargets="CoreCompile">
```

Comment:
This is a hack to remove `System.Net.Security.dll` dependency brought over all the way from NetStandardLibrary.
Here's the problem: This test project is doing unit tests by compiling some of the actual source files from `System.Net.Security` and the assumption is there will be no reference to `System.Net.Security` contract. However, in netstandard1.7, NetStandardLibrary causes a dependency to that. The hack enables us to work around the conflict errors at compile time.